bin/shell/osh.pl

```diff
@@ -875,7 +875,13 @@ sub main_exit {
             # use system() instead of OVH::Bastion::execute() because we need it to grab the term
             my $pamtries = 3;
             while (1) {
-                my $pamsysret = system('pamtester', 'sshd', $sysself, 'authenticate');
+                my $pamsysret;
+                if (OVH::Bastion::is_freebsd()) {
+                    $pamsysret = system('sudo', '-n', '-u', 'root', '--', '/usr/bin/env', 'pamtester', 'sshd', $sysself, 'authenticate');
```

Comment:
Maybe to move inside a pamtester helper function

Code is the same at line 1316

Reply:
done